rosidl_generator_py/resource/_msg.py.em

```diff
@@ -292,16 +453,33 @@ if isinstance(type_, AbstractNestedType):
 ,  # noqa: E501
 @[end for]@
     )
+@{
+# Taken from flake8-builtins
+# https://github.com/gforcada/flake8-builtins/blob/689ad01c03cb52ae73be23d19706e6a4a491f4e9/flake8_builtins.py
+import inspect
+import builtins
+BUILTINS = [
+    a[0]
+    for a in inspect.getmembers(builtins)
+]
+}@
 
-    def __init__(self, **kwargs):
-        if 'check_fields' in kwargs:
-            self._check_fields = kwargs['check_fields']
+    def __init__(self,
```

Comment:
Please add `*,` here to force the arguments to be keyword only to avoid downstream code relying on the order of fields, and breaking if the message fields are added or reordered.

```suggestion
    def __init__(self, *,
```